t/food.t

```diff
@@ -93,7 +93,7 @@ my $product_ref = {
 
 special_process_product($product_ref);
 
-ok( (not has_tag($product_ref, 'categories', 'en:unsweetened-beverages')), 'should not add en:unsweetened-beverages' ) || diag explain $product_ref;
+#ok( (not has_tag($product_ref, 'categories', 'en:unsweetened-beverages')), 'should not add en:unsweetened-beverages' ) || diag explain $product_ref;
```

Comment:
Are these tests outdated?

Reply:
They are outdated if we don't turn off and on automatically the categories for sweetened and unsweetened beverages using the analysis of the ingredients. We used to do it automatically, but it's causing issues, that's why I made the changes in Food.pm.  But if we do reenable it at some point, then the tests would be useful, that's why I commented them.

Reply:
You can always get the code back from the git history. A huge problem with commented code is, that no one knows (from the file itself) why the code is commented. Come back a few months later, and you probably won't know why those single lines are comments. Additionally, dead code makes the source files difficult to read.